tests/rt/devel_packages.pm

```diff
@@ -0,0 +1,26 @@
+# SUSE's openQA tests
+#
+# Copyright Â© 2009-2013 Bernhard M. Wiedemann
+# Copyright Â© 2012-2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+use base "opensusebasetest";
+use strict;
+use testapi;
+
+# Installation media contain devel packages
+# https://fate.suse.com/316652
+sub run() {
+    #script_run "zypper lr -u > /dev/$serialdev";
+    my $repo = "SLERT12-SP1_12.1-0";
+    my @packages = ("babeltrace-devel", "lttng-tools-devel", "kernel-compute-devel", "kernel-compute_debug-devel", "kernel-rt-devel", "kernel-rt_debug-devel", "kernel-devel-rt", "libcpuset-devel");
+    foreach my $pkg (@packages) {
+        assert_script_run "zypper -q search -r $repo $pkg";
```

Comment:
Are you sure within openQA it makes sense to check every individual package by itself? This could be very fragile, e.g. when test dependencies change we need to adjust this not knowing what to expect. IMHO from a user point of view this test does not make so much sense in this way. What does the user really want to achieve?